tools/ci_build/github/apple/default_full_ios_training_framework_build_settings.json

```diff
@@ -0,0 +1,23 @@
+{
+    "build_osx_archs": {
+        "iphoneos": [
+            "arm64"
+        ],
+        "iphonesimulator": [
+            "arm64",
+            "x86_64"
+        ]
+    },
+    "build_params": [
+        "--ios",
+        "--parallel",
+        "--use_xcode",
+        "--enable_training_apis",
+        "--build_apple_framework",
+        "--use_coreml",
```

Comment:
training does not support `--use_coreml` and `--use_xnnpack`. So, users my not be able to train with coreml or xnnpack.
Having said that, the user may want to run inferencing with coreml after locally training their model with CPU.

Is this file mostly used as an example, or will this be used for the packaging build (for the next release)?

Reply:
I was mainly using it as an example. For release, I am not sure if we should include `--use_coreml` and `--use_xnnpack`

Reply:
In that case, let's remove both `--use_coreml` as well as `--use_xnnpack` from these build params.

Reply:
note that as is, the packaging pipeline is using this file and it will be used to produce release packages.